docs/source/peer_ledger_snapshot.md

```diff
@@ -0,0 +1,164 @@
+* **The importance of querying old data**. Some channels and use cases might call for querying old blocks regularly, while in other channels it might only be necessary to know the current state. This must be balanced by the server costs experienced by your organization.
+
+## Overview
+
+Snapshots can be used by organizations that already have peers on a channel or by organizations new to a channel. Whatever the use case, the process is largely the same.
+
+If the organization is new to the channel, the organizations currently joined to the channel must:
+
+1. **Schedule a snapshot**. These snapshots must be taken at **exactly the same ledger height** on each peer. This will allow the new peer organization to evaluate the snapshots to make sure they contain the same data. This ledger height must be equal or higher than the current ledger height. They cannot be taken from a lower ledger height. If you attempt to schedule a snapshot at a height lower than the current height you will get an error. Note that a peer that already used a snapshot to join a channel can also be used to take a snapshot.
+2. **When the ledger height is reached, the snapshot is taken by the peer**. The snapshot is comprised of a folder that includes files that contain the public state, hashes of private state, transaction IDs, and the collection config history. A file containing metadata relating to these files is also included. For more information, check out [contents of a snapshot](#contents-of-a-snapshot).
+2. **The snapshot is sent to the new organization**. This must be completed out of band. Because snapshot files are not compressed, it is likely that peer administrators will want to compress these files before sending them. In a typical scenario, the administrator will receive the snapshot from one of the existing organizations but will want to receive the snapshot metadata from more than one organizations in order to verify the snapshot received.
+
+The new organization will then:
+
+1. **Evaluate the snapshots from the other organizations to ensure they contain the same data**. An administrator of the peer organization attempting to use the snapshot to join the peer to the channel should independently compute the hashes of the snapshot files and match these with the hashes present in the metadata file. In addition, the administrator may want to match the metadata files from more than one organization, depending on the trust model established by the network. In some scenarios, the administrator may want the administrators of other organizations to sign the metadata file for its records.
+2. **Join the peer to the channel using the snapshot**. When the peer has finished joining the channel using the snapshot, it will begin pulling private data according to the collections it is a member of.
```

Comment:
```suggestion
2. **Join the peer to the channel using the snapshot**. When the peer has finished joining the channel using the snapshot, it will begin pulling private data according to the collections it is a member of. It will also start committing blocks as normal, starting with any blocks greater than the snapshot height that are available from the ordering service.
```